### PR TITLE
docs: fixing dead link

### PR DESCRIPTION
**Pull request by Salanoid**
Fixing guide dead link for issue #57067.

## PR Type
What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: [57067](https://github.com/angular/angular/issues/57067)

### CODE REVIEW

File: adev/src/content/guide/zoneless.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ These notifications include:
 ### `OnPush`-compatible components
 
 One way to ensure that a component is using the correct notification mechanisms from above is to
-use [ChangeDetectionStrategy.OnPush](../best-practices/skipping-subtrees#using-onpush).
+use [ChangeDetectionStrategy.OnPush](/adev/src/content/best-practices/runtime-performance/skipping-subtrees.md#using-onpush).
```
Comment (JeanMeche) [inline adev/src/content/guide/zoneless.md:45]: There are no extensions in our urls. 
```suggestion
use [ChangeDetectionStrategy.OnPush](/adev/src/content/best-practices/runtime-performance/skipping-subtrees#using-onpush).
```

**Comment by JeanMeche** [comment]
Also can you make sure to squash the commits. We'd like this change to be a single commit. Thank you.

**Review by JeanMeche — APPROVED** [review]
Thanks for this !

**Comment by JeanMeche** [comment]
It seems that you authored your commit with 2 different accounts. 
Could you please signe the CLA with _silv*******jan​@cubus.ro_, thank you.

**Comment by atscott** [comment]
This PR was merged into the repository by commit aeaf07d7cd8186653cde6eacc9e560f03c9dfd98.

The changes were merged into the following branches: main, 18.1.x

**Pull request closed without merging**